dshr/dshr_mod.F90

```diff
@@ -238,36 +236,31 @@ subroutine dshr_mesh_init(gcomp, sdat, nullstr, logunit, compname, model_nxg, mo
 
     ! local variables
     type(ESMF_VM)                  :: vm
+    logical                        :: masterproc
```

Comment:
Seems that we are getting away from using this name - replace with mainproc?

Reply:
I think this needs to be a global replace - since we need to also change master_task everywhere. Do you want me to do this in this PR. I could do that. Let me know.